docs/environments/macvrp.md

```diff
@@ -0,0 +1,48 @@
+
+A new problem instance is generated by resetting the environment. The problem instance contains
+coordinates for each node sampled from a uniform distribution inside the map boundries, and each node
+(except for depot) has a specific demand which is an integer value sampled from a uniform
+distribution between 1 and the maximum demand.
+The number of nodes with demand is a parameter of the environment.
+
+## Observation
+Each agent receives information on the coordinates, demands, time windows and penalty coefficients of all the customer nodes. Futhermore the agents receive positions, local times and vehicle capacity information on all vehicles. Lastly an action mask is also provided to each agent.
+
+**Observation Spec**:
+
+- **node_coordinates**: jax array (float32) of shape (num_vehicles, num_customers + 1, 2), shows an array of the coordinates of each customer node
+and the depot node.
+- **node_demands**: jax array (int16) of shape (num_vehicles, num_customers + 1,), shows an array of the demands of each city node
+(and depot node where the demand is set to 0).
+- **node_time_windows**: jax array (float32) of shape (num_vehicles, num_customers + 1, 2), shows an array of the early and late time cutoffs for each customer.
+- **node_penalty_coefs**: jax array (float32) of shape (num_vehicles, num_customers + 1, 2), shows the early and late penalty coefficients for arriving early or late at a customer's location.
+- **other_vehicles_position**: jax array (int16) of shape (num_vehicles, num_vehicles - 1), shows the positions of all other vehicles.
+- **other_vehicles_local_times**: jax array (float32) of shape (num_vehicles, num_vehicles - 1), shows the local times of all other vehicles.
+- **other_vehicles_capacities**: jax array (int16) of shape (num_vehicles, num_vehicles - 1), shows the capacities of all other vehicles.
+- **vehicle_position**: jax array (int16) of shape (num_vehicles), shows the positions of the vehicles controlled by the agents.
+- **vehicle_local_time**: jax array (float32) of shape (num_vehicles), shows the local times of the vehicles controlled by the agents.
+- **vehicle_capacity**: jax array (int16) of shape (num_vehicles), shows the capacity of the vehicles controlled by the agents.
+- **action_mask**: jax array (bool) of shape (num_vehicles, num_customers + 1,), denoting which actions are possible (True) and
+which are not (False).
```

Comment:
Could we respect the same format as the other environments for this, please? See e.g. cvrp.md

Reply:
Updated :+1: